### DISCUSSION TITLE
Command to open tmux in new session and name session to working dir

### DISCUSSION THREAD

**Discussion by Hackman78** [gokcehan/lf#1723] [Show and tell]
```
map T ${{
    cd $f
    tmux new -s $(basename $f)
}}
```
It's not much but it's honest work. Next I want to write one where it checks if i have a curr tmux session open and opens it if exists. If anybody has any ideas lmk

**Comment by DusanLesan** [comment]
Something like this?
```shell
map T ${{
	if tmux has-session -t $id 2>/dev/null; then
		notify-send "Session already exists"
		tmux attach-session -t $SESSION_NAME
	else
		notify-send "Creating new session"
		tmux new-session -s $id
	fi
}}
```

**Reply by Hackman78** [reply]
```
map T ${{
    cd $f
    if tmux attach-session -t $(basename $f); then
else
    tmux new -s $(basename $f)
fi
}}
```
I think I finally settled on this but I like this too ill try that rn

**Reply by Hackman78** [reply]
apparently Macos doesn't have notify-send smh

**Reply by DusanLesan** [reply]
Oh that is unfortunate even though it was here only for debugging. I like putting notify-send everywhere while making new stuff. You might be able to log to a file and read it elsewhere.

**Reply by Hackman78** [reply]
I was just using echo with $f if it was working it'd be ok and if it didn't work I'd see what the var I was looking for was
